app/locale/en_US.json

```diff
@@ -48,6 +48,9 @@
     "AN_APP_END_CENSORSHIP": "An app to end censorship",
     "THANKS_FOR_DOWNLOADING": "Thanks for downloading!",
     "END_CENSORSHIP_DESCRIPTION": "Internet censorship hurts billions, but we can stop it. Lantern is the app you install on your computer that lets the whole world escape censorship.",
+    "USER_PRAISE_FROM_SCMP": "“Lantern is the most intuitive tool to access the internet freely”",
+    "USER_QUOTED": "- Lantern user quoted in the",
+    "SCMP": "South China Morning Post",
```

Comment:
Awesome that you created translation strings for the press quotes already. I'll go ahead and push these to Transifex now.